job-server/src/spark.jobserver/JobInfoActor.scala

```diff
@@ -11,7 +11,7 @@ import spark.jobserver.io.{JobDAO, JobInfo}
 
 object JobInfoActor {
   // Requests
-  case class GetJobStatuses(limit: Option[Int])
+  case class GetJobStatuses(limit: Option[Int],status: Option[String])
```

Comment:
I think we should borrow style config from spark project. Also scalariform settings.